keps/sig-api-machinery/4153-declarative-validation/README.md

```diff
@@ -1212,6 +1213,28 @@ See [Unit Test Instrumentation](#unit-test-instrumentation) for unit testing pla
 - `k8s.io/apiserver/pkg/cel/library`: `10/02/2023` - `69`
 - `k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/model`: `10/02/2023` - `83.3`
 
+##### Runtime verification testing
+
+In addition to unit and fuzz tests, we will offer a means of running declarative validation in a "shadow mode"
+such that the presence of mismatches between declarative validation and hand written validation can
+be safely checked against production workloads.
+
+When a `DeclarativeValidationShadowMetrics` feature gate is enabled, the following will be collected for each validation operation:
```

Comment:
Talked offline, might make sense to update the name suggested there: DeclarativeValidationMismatchMetrics